modules/apps/frontend-theme/frontend-theme-classic/src/WEB-INF/frontend-token-definition.json

```diff
@@ -521,6 +521,19 @@
 							"name": "btnOutlineSecondaryBorderColor",
 							"type": "String"
 						},
+						{
+							"defaultValue": "#272833",
```

Comment:
Just curious if this is the right color for `btn-outline-secondary`, I think I might have broke it during the Bootstrap 4 take over. It used to be `#6B6C7E` before that pr I think.

Reply:
Wow, we have a lot of `#272833` in the `frontend-token-definition.json`.
We can adjust this in other PR